Annotato/Annotato/Views/Document/List/DocumentListCollectionCellView.swift

```diff
@@ -48,4 +50,18 @@ class DocumentListCollectionCellView: UICollectionViewCell {
         label.bottomAnchor.constraint(equalTo: self.bottomAnchor).isActive = true
         label.centerXAnchor.constraint(equalTo: self.centerXAnchor).isActive = true
     }
+
+    private func addTapGestureRecognizer() {
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didTapOnCellView))
+        addGestureRecognizer(gestureRecognizer)
+    }
+
+    @objc
+    private func didTapOnCellView() {
+        guard let document = document else {
+            print("No document in this cell view")
```

Comment:
Shall we remove this print statement? We can show an error to the user easily by conforming to `AlertPresentable`, we can also log the error.

Reply:
I removed it